modules/apps/segments/segments-web/src/main/resources/META-INF/resources/field/select_organizations.jsp

```diff
@@ -46,6 +46,15 @@ SelectOrganizationsDisplayContext selectOrganizationsDisplayContext = (SelectOrg
 			keyProperty="organizationId"
 			modelVar="organization"
 		>
+			<%
```

Comment:
The JS in `select_organizations.jsp` and `select_users.jso` is pretty much the same thing. Couldn't we extract that and leave it as piece that we can use when some other entity needs to be _multiselected_.

Reply:
I didn't think it was too concerning at the time, but to keep these js blocks synced I'll create a shared `.jspf` file that both `select_*.jsp` will include.